File: js/1180.js
var input = require('fs').readFileSync('./dev/stdin', 'utf8')
var lines = input.split('\n')
n = lines.shift()

array = lines.shift()

array = array.split(' ').map(x => parseInt(x))
x = array
let menor = x[0]
let pos = 0

for (i=1;i<n;i++){
    if(x[i]<menor){
        menor=x[i]
        pos=i
    }
}

console.log(`Menor valor: ${menor}`)
console.log(`Posicao: ${pos}`)

File: python/PROBLEMS/1180.py
n = int(input())
x = input().split(' ')

pos = 0
menor = int(x[0])

for i in range(1, n):
    try:
        num = int(x[i])
        if num < menor:
            menor = num
            pos = i
    except ValueError:
        pass

print(f"Menor valor: {menor}")
print(f"Posicao: {pos}")
